### ISSUE TITLE
Project can't be compiled as ap4k deps has not been released for 1.0.2

### ISSUE THREAD

**Issue by cmoulliard** [ap4k/ap4k#6]
Project can't be compiled as ap4k deps has not been released for 1.0.2

```
[ERROR] Failed to execute goal on project ap4k-core: Could not resolve dependencies for project io.ap4k:ap4k-core:jar:1.0-SNAPSHOT: Failure to find io.ap4k:ap4k-dependencies:jar:1.0.2 in http://repo1.maven.org/maven2/ was cached in the local repository, resolution will not be reattempted until the update interval of Maven Central has elapsed or updates are forced -> [Help 1]
[ERROR] 
```

**Comment by cmoulliard** [comment]
Can you update the ticket please @iocanel ?

**Comment by iocanel** [comment]
Versions 1.0.1 and 1.0.2 never made it to central, due to project misconfiguration.

Fixed the issues and also set version to 0.1.0 to reflect the project maturity. Release is to central now, ap4k has been configured to use 0.1.0 and we can safely close this one.
